spec/teamsnap_spec.rb

```diff
@@ -262,28 +262,4 @@
       )
     end
   end
-
```

Comment:
I couldn't figure out a good way to cause a 500 on the server. I figured any 500 error should be fixed. The below test stopped working because we fixed the 500 that was causing it.